indico/modules/events/contributions/models/contributions.py

```diff
@@ -332,6 +332,21 @@ def __table_args__(cls):
         )
     )
 
+    @declared_attr
+    def paper_last_revision(cls):
+        # Incompatible with joinedload
+        return db.relationship(
+            'PaperRevision',
+            uselist=False,
+            lazy=True,
+            viewonly=True,
+            primaryjoin=(db.and_((PaperRevision.contribution_id == cls.id),
```

Comment:
no need for the innermost `()` now; they were only needed because of the precedence of `&`